index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+
+    <!-- Header -->
+    <header>
+        <div class="header-container">
+            <a href="/" class="logo">
+                <img src="assets/Property 1=sm@3x.png" alt="판다마켓 로고">
+            </a>
+            <a href="/login.html" class="login-btn">
+                <img src="assets/type=solid, status=default.png" alt="로그인 버튼">
```

Comment:
❗️ 수정요청
이미지가 아닌 html로 구현하시는 것을 추천드려요~

```suggestion
                로그인
```